### DISCUSSION TITLE
Multiple existing boards support

### DISCUSSION THREAD

**Discussion by jgauchia** [jgauchia/IceNav-v3#193] [Ideas]
Hello! My idea is to provide support for this firmware on different boards available on the market. If you want IceNav to work on them, please let me know which model you have, and I will try to add support. Thank you!

**Comment by makergamer** [comment]
The [Cheap Yellow Display](https://github.com/witnessmenow/ESP32-Cheap-Yellow-Display) should be considered a strong target candidate given all the built-in components (touch screen, sdcard, lipo connection & charging circuit, etc.). It has a JST for the i2c sensor. GPS is just a serial connection away. So basically a CYD plus 2 add-ons. 

You'll find its Discord channel to be very active & with a large supportive community.

**Reply by jgauchia** [reply]
Hi, yes, 
I'll try to get one or with datasheet, specifications try to implement It, no problem.

Thank you!
